### DISCUSSION TITLE
array_key_exists(): The first argument should be either a string or an integer

### DISCUSSION THREAD

**Discussion by gvillela7** [spatie/laravel-activitylog#941] [Q&A]
Hi, I installed LogsActivity in my project and I'm only using it on the Model I need. When I try to update a record, I get the following error:
array_key_exists(): The first argument must be a string or an integer.

I followed all the installation steps.
Can you help?
"spatie/laravel-activitylog": "^3.17",

**Comment by Gummibeer** [comment]
Hey,
we would need more details to help you here. A full stacktrace, your code resulting in the error and so on.
